MODEL-GUIDE.md

```diff
@@ -135,6 +135,20 @@ The build script will enforce the following keyword conventions:
             enum: [current, deprecated, obsolete, under_review]
             default: current
   ```
+* `x-status-message`
```

Comment:
An alternate approach would be to change the x-status extension from an enum to an object to allow for more flexibility in the future.
```yaml
x-status:
  type: object
  properties:
    status:
      type: string
      enum: [current, deprecated, obsolete, under_review]
    additional_information:
      type: string
      description: Use this property to provide additional information about the status property.
```

Reply:
yeah agreed, I will change accordingly

Reply:
updated as per the comments @ajbalogh